src/foam/nanos/controller/ApplicationController.js

```diff
@@ -60,6 +60,7 @@ foam.CLASS({
     'appConfig',
     'as ctrl',
     'currentMenu',
+    'filteredTableColumns',
```

Comment:
You don't want this on the ApplicationController. This ties the ApplicationController to one set of filteredTableColumns but there's no reason we can't render multiple table views at once. I think troy's suggestion was to put this on `foam.comics.DAOController`